### PR TITLE
Python3 fixes, add information on testing

### PR DESCRIPTION
**Pull request by barseghyanartur**
Python3 fixes, add information on testing.

### CODE REVIEW

File: elasticmock/fake_elasticsearch.py [hunk 1]
```diff
@@ -216,7 +216,7 @@ def suggest(self, body, index=None, params=None):
             raise NotFoundError(404, 'IndexMissingException[[{0}] missing]'.format(index))
 
         result_dict = {}
-        for key, value in body.iteritems():
+        for key, value in body.items():
```
Comment (vrcmarcos) [inline elasticmock/fake_elasticsearch.py]: This change will break Python 2.7 code. Could you verify if you're running into Python 3 or Python 3 before?

File: tests/test_elasticmock.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-from unittest import TestCase
+import unittest
```
Comment (vrcmarcos) [inline tests/test_elasticmock.py]: This modification doesn't seem necessary. Is there a reason to do this?

File: tests/test_elasticmock.py [hunk 3]
```diff
@@ -206,3 +206,7 @@ def test_should_return_suggestions(self):
                 }
             ],
         }, suggestion)
+
+
+if __name__ == '__main__':
```
Comment (vrcmarcos) [inline tests/test_elasticmock.py]: You don't need to do this to run the tests. You can run `make test` from the root folder :)

File: elasticmock/fake_elasticsearch.py (hunk 1 above)
Comment (barseghyanartur) [inline elasticmock/fake_elasticsearch.py]: `items` work on both Python3 and Python2. Considering the fact, that body is never that big, it's OK to use just `items`.

https://stackoverflow.com/questions/10458437/what-is-the-difference-between-dict-items-and-dict-iteritems

File: tests/test_elasticmock.py (hunk 3 above)
Comment (barseghyanartur) [inline tests/test_elasticmock.py]: The way I changed it, will work on a broader number of test runners.

File: elasticmock/fake_elasticsearch.py (hunk 1 above)
Comment (vrcmarcos) [inline elasticmock/fake_elasticsearch.py]: Awesome! Didn't know that. Thanks!

File: tests/test_elasticmock.py (hunk 3 above)
Comment (vrcmarcos) [inline tests/test_elasticmock.py]: Nice! I will merge this as soon as possible :) Thanks!

**Comment by manalelidrissi** [comment]
Hello, when I was introducing elasticmock for unit testing, I found out that it doesn't support the "must" and "should" features of Elasticsearch_dsl. Have anyone experienced this issue ? If so, What do you advise me to do ?

**Comment by barseghyanartur** [comment]
@manalelidrissi:

From my experience, you should better stick to testing with real ES instances.

**Comment by manalelidrissi** [comment]
@barseghyanartur 

Could you please give more details?

ES instances don't provide a way to perform queries and filters, it can only be done using ES_dsl query. must, should, must_not, match_all can't be performed and I'm stuck in it. I would be grateful if you could help me move forward.

**Pull request merged**